utility_analysis_new/poisson_binomial.py

```diff
@@ -51,8 +66,16 @@ def compute_pmf_approximation(mean: float, sigma: float, skewness: float,
     The computation is based on paper chapter 3.3 (refined normal approximation)
     https://www.researchgate.net/publication/257017356_On_computing_the_distribution_function_for_the_Poisson_binomial_distribution
```

Comment:
Can we add the new assumption to the doc?

E.g. values with negligible probabilities will be dropped for efficiency reasons. The total probability for the dropped values is no more than 10^(-15).